lib/resolvers/project.js

```diff
@@ -1,7 +1,8 @@
-const resolver = require('./base-resolver');
 const { pick, get, omit, isEqual, isInteger } = require('lodash');
-const { generateLicenceNumber } = require('../utils');
 const moment = require('moment');
+const { BadRequestError } = require('@asl/service/errors');
```

Comment:
This isn't an http service so this is redundant, easier just to `throw new Error`.

Not even sure why it's in package.json because it's not used anywhere I could find.

Reply:
```suggestion
```